changedetectionio/content_fetchers/playwright.py

```diff
@@ -172,7 +172,7 @@ def run(self,
 
             # Run Browser Steps here
             if self.browser_steps_get_valid_steps():
-                self.iterate_browser_steps()
+                self.iterate_browser_steps(url)
```

Comment:
try to use `start_url = url` or something

Reply:
Done.